### PR TITLE
Added "getBlockCount" Webservice

### PR DESCRIPTION
**Pull request by osietrz**
Added webservice that fetches the current total block count of the network from the electraExplorer api.
Useful when node is syncing to show how many block are left to download.

### CODE REVIEW

File: src/web-services/getBlockCount.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+/**
+ * Get the current total block count from the ElectraExplorer API.
+ * Useful when syncing to get the highest current block on network
+ */
+export default async function(): Promise<number> {
+  const [ err, res ] = await to(Axios.get<number>(URI))
+  if (err) throw new Error(`webServices#getBlockCount(): ${err.message}`)
+
+  if (res === undefined || typeof res !== 'number') {
+    throw new Error(`webServices#getBlockCount(): We did't get the expected response from ElectraExplorer.`)
```
Comment (Hyjaz) [inline src/web-services/getBlockCount.ts]: small type `didn't` instead of `did't`

File: src/web-services/index.ts [hunk 2]
```diff
@@ -1,14 +1,17 @@
 import getBalanceFor from './getBalanceFor'
 import getCurrentPriceIn, { CoinMarketCapCurrency } from './getCurrentPriceIn'
+import getBlockCount from './getBlockCount'
 
 export { CoinMarketCapCurrency }
 
-export interface WebServices {
+export interface WebServices {
```
Comment (Hyjaz) [inline src/web-services/index.ts]: remove extra space

File: src/web-services/index.ts [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
-  getCurrentPriceIn: typeof getCurrentPriceIn
+  getCurrentPriceIn: typeof getCurrentPriceIn  
+  getBlockCount: typeof getBlockCount  
 }
 
 export default {
   getBalanceFor,
-  getCurrentPriceIn
+  getCurrentPriceIn,
+  getBlockCount
```
Comment (Hyjaz) [inline src/web-services/index.ts]: Order alphabetically?

File: src/web-services/index.ts [hunk 4]
```diff
@@ -1,14 +1,17 @@
...
+import getBlockCount from './getBlockCount'
 
 export { CoinMarketCapCurrency }
 
-export interface WebServices {
+export interface WebServices {  
   getBalanceFor: typeof getBalanceFor
-  getCurrentPriceIn: typeof getCurrentPriceIn
+  getCurrentPriceIn: typeof getCurrentPriceIn  
+  getBlockCount: typeof getBlockCount
```
Comment (Hyjaz) [inline src/web-services/index.ts]: Order alphabetically.

File: src/web-services/index.ts [hunk 5]
```diff
@@ -1,14 +1,17 @@
 import getBalanceFor from './getBalanceFor'
 import getCurrentPriceIn, { CoinMarketCapCurrency } from './getCurrentPriceIn'
+import getBlockCount from './getBlockCount'
```
Comment (Hyjaz) [inline src/web-services/index.ts]: Order alphabetically

File: src/web-services/getBlockCount.ts [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import to from 'await-to-js'
+import Axios from 'axios'
```
Comment (Hyjaz) [inline src/web-services/getBlockCount.ts]: Order alphabetically

**Comment by osietrz** [comment]
@ivangabriele You added a way to check for highest blockcount among connected peers, so this method isn't interesting for hard wallets, but it's still interesting for light wallets, to see where the block tip is currently at.

**Pull request closed without merging**